activerecord/test/cases/adapters/postgresql/connection_test.rb

```diff
@@ -90,7 +90,7 @@ def test_reset_with_transaction
     end
```

Comment:
These tests were added by [this](https://github.com/rails/rails/pull/6386). I think we should keep these test cases.